test/test_autograd.py

```diff
@@ -995,6 +995,15 @@ def test_setitem_mask(self):
         self._test_setitem_tensor((5, 5), Variable(mask))
         self._test_setitem_tensor((5,), Variable(mask[0]))
 
+    def test_select_sum(self):
+            x = Variable(torch.randn(10), requires_grad=True)
+
+            def func(x):
+                return x.select(0, 1).sum()
+
+            gradcheck(func, [x])
+            gradgradcheck(func, [x])
```

Comment:
Can you add a comment for this test? I don't know what makes a combination of select and sum special